### DISCUSSION TITLE
Increase liquidity if we run the service with docker container

### DISCUSSION THREAD

**Discussion by tipogi** [ACINQ/phoenixd#35] [Q&A] [answered]
Following the issue [#31](https://github.com/ACINQ/phoenixd/issues/31) that I open (finally it was not an issue... 🙏), how can I increase the liquidity because with the official container it is not possible.

The `.phoenix/phoenix.conf` file, it is owned by __root__ and the user of the container is __phoenix__ so it is not possible to edit that file and threfore we cannot add/edit that param (`auto-liquidity=5m`). Which would it be the way to add more liquidity in that channel? The first ideia that comes to my mind is:
- Close the channel: `phoenix-cli closechannel`
- Delete the container and the image
- Edit the docker file, _ENTRYPOINT [..., "--auto-liquidity", "5m"]_
- Build a new image and create a new container

**Comment by pm47** [comment, marked as answer]
This is a docker question, not a phoenixd question, please stay on topic.

I guess you could do this: `docker run <image> --auto-liquidity=5m`

**Reply by tipogi** [reply]
Yes thats right, closing the question
